node/src/blockchain/blockchain_interface.rs

```diff
@@ -254,12 +276,22 @@ where
             )
             .build();
 
-        let log_request = self.web3.eth().logs(filter);
+        let end_block_number = match Option::from(end_block) {
```

Comment:
What name do you recommend instead of `end_block_number`? It's really a fallback value that is used in the event there's an error in retrieving the latest block number from the chain. This value may also be changed after GH-585 where the blockchain service might provide a maximum block count delta from the start block as a limitation we have to use on subsequent calls.

Reply:
Actually `fallback_end_block_number` or somehow shorter looks good to me. (I know you've got better sense for the language, so I'm not suggesting nothing but that the word `fallback` would be nice to have in there). What do you think? It might stop making people wonder about what that is.